src/Stryker.Core/Stryker.Core/Mutators/ConditionStatementMutator.cs

```diff
@@ -0,0 +1,61 @@
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.Mutants;
+
+namespace Stryker.Core.Mutators
+{
+    public class ConditionStatementMutator : MutatorBase<ExpressionSyntax>, IMutator
+    {
+        public override IEnumerable<Mutation> ApplyMutations(ExpressionSyntax node)
+        {
+            if (ContainsArguments(node)) yield break;
```

Comment:
We don't use short form if's

Reply:
valid point, remember it from last time, thanks